docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
+= Get started with security in Quarkus by using Basic authentication
 
```

Comment:
@sberyozkin - After a discussion with Erin, we agreed that this title nicely follows the Diataxis and preferred Quarkus doc style.

@ebullient - Note that I removed the 'and JPA' bit in my originally suggested title to incorporate Siarhei's previous comment about JPA.